chunky/src/java/se/llbit/chunky/resources/Image.java

```diff
@@ -0,0 +1,31 @@
+package se.llbit.chunky.resources;
+
+/**
+ * Generic image giving read only access
+ */
+public interface Image {
```

Comment:
👍 for this interface!
Maybe we could reduce it even more so that `Texture` can implement it too?

Reply:
Nope, bad idea. A texture might contain multiple textures internally later (for normal maps, specular maps, metalness maps).